planemo/workflow_lint.py

```diff
@@ -420,15 +444,21 @@ def _lint_dockstore_workflow_entry(
             lint_context.error(f"{DOCKSTORE_REGISTRY_CONF} workflow entry missing required key {required_key}")
             found_errors = True
 
-    for recommended_key in ["testParameterFiles"]:
+    if lint_context.iwc_grade:
```

Comment:
So far we only enable / disable linters with command line flags, but we do not change the linter level. Instead we allow the user to change the fail/report level.

Maybe we should leave it this way .. for now.

Reply:
My issue is that workflow_lint tend to give a lot of warnings which are not true like: `WARNING: Input Input gtf of workflow step 0 is disconnected.` so we cannot make it fails on WARNING... But maybe this is not a good reason...

Reply:
Yep, this seems to be a bug (not sure) that should be fixed.

Reply:
> Maybe we should leave it this way .. for now.
You mean that you accept this change or I should revert these changes?

Reply:
I would just make it an error and not run it if the `--iwc` flag is not given.